azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +34,16 @@
   QUEUED(110),
   FAILED_SUCCEEDED(120),
   CANCELLED(130);
+  // status is TINYINT and in H2 DB the possible values are: -128 to 127
+  // so trying to store CANCELLED in H2 fails at the moment
+
+  private static final Map<Integer, Status> numValMap = new HashMap<>();
```

Comment:
what do you think of using ImmutableMap https://google.github.io/guava/releases/21.0/api/docs/com/google/common/collect/ImmutableMap.html?